src/widgets/FileWidget/index.js

```diff
@@ -183,9 +206,18 @@ const getProps = (props) => {
   if (adjustedDownloadBasepath === undefined) {
     adjustedDownloadBasepath = '';
   }
+  if (!adjustedRemoveComponent) {
+    adjustedRemoveComponent = RemoveHandle;
+  }
   if (!adjustedUploadComponent) {
     adjustedUploadComponent = UploadHandle;
   }
+  if (adjustedRemoveStyle === undefined) {
+    adjustedRemoveStyle = null;
+  }
+  if (adjustedRemoveLabel === undefined) {
+    adjustedRemoveLabel = 'Delete';
+  }
```

Comment:
Please move the `if` for `adjustedUploadComponent` here to be grouped with the other `if`s.